### DISCUSSION TITLE
LMWG support for Earth System Prediction

### DISCUSSION THREAD

**Discussion by wwieder** [ESCOMP/CTSM#2116] [General]
The LWMG would like to support new application of the land model in emerging model configurations.  This includes:
- **High resolution** configuration that are used for numerical weather prediction ([NWP](https://ral.ucar.edu/nsap/numerical-weather-prediction-nwp-and-data-assimilation)), [SIMA](https://sima.ucar.edu/), [EarthWorks](https://hogback.atmos.colostate.edu/earthworks/), [DYAMOND experiments](https://progearthplanetsci.springeropen.com/articles/10.1186/s40645-019-0304-z), etc;
- **Initialized predictions,** like those used by [SMYLE](https://www.cesm.ucar.edu/working-groups/earth-system/simulations/smyle-real-time) & crated by the [ESPWG](https://www.cesm.ucar.edu/working-groups/earth-system)); and 
- **Impact studies**, which may rely on fidelity of land model output.

Following conversations at the [ESPAT workshop](https://ncar.ucar.edu/predictability), this seems like a growth area for NCAR and one that the LWMG would like to support.

Broadly, this discussion is to support simulations which requires creating high resolution surface datasets and/or "non-standard" initial condition.  This is Discussion Board is intended to track these efforts as we develop expertise and tools in this area.  See also [Project Board #31](https://github.com/orgs/ESCOMP/projects/31)

-----
This thread first came up when several of us have been having isolated conversations, so I wanted to start a general discussion here to organize what the LMWG needs to do to help support DYAMOND (3.75km atmosphere & land) and SIMA (high resolution over CONUS for recent NW heat dome and Texas freeze events).

I'm less clear on a timeline for this, although it seems like it's more on the order of weeks than months.  For now I just wanted to start a conversation and put this on everyone's radar.
 
Generally it seems like we need to provide:

**1. Surface datasets:**
  - Present day surface dataset for the (0.25 degree?) land only run used in (2) for generating land initial conditions with ERA5.
  - 3.75 km for the DYAMOND run (that may already be available from EarthWorks or @fischer-ncar?)
  - Variable resolution for the SIMA runs focusing on CONUS. 

**2. Land Initial Conditions:**
  - For now we've discussed generating this from CTSM5.1sp simulations forced with high resolution ERA5 reanalysis.
  - This simulation could likely start with initial conditions from a 1 degree CTSM5.1sp-GSWP3 forced run?
  - @briandobbins can tell us what start date(s) are needed for the experiments.

@briandobbins does this seem accurate?
@dlawrenncar suggested that showing we can provide these data may ultimately take some pressure off of CGD / CESM.

If so, I'd suggest the following: 
- @ekluzek and @slevis-lmwg may be able to help with surface dataset needs?  These don't need to be from CTSM5.2, but do the new mksrfdata_esmf tools make things easier?
- @swensosc, can you create datm data from ERA5 files that @adamrher may already have on disk?
- @olyson, after these are done, can you run a year or two of spinup with these ERA5 datm files (at the ERA5 resolution).

**Comment by ekluzek** [comment]
I'm thinking that the CTSM5.2 branch should be used for making surface datasets. I think you'll run into a ton of problems trying to creating the mapping datasets as well as run mksurfdata_map for these ultra high resolution cases. Yes, @slevis-lmwg and I can help with that if needed.

**Comment by fischer-ncar** [comment]
I've been able to run I2000Clm50SpRs at 3.75km on derecho with 224 nodes and under subscribing with 14,336 tasks.  But I haven't tried running any F compsets.  Also, this is with CTSM5.2 and a memory fix from Mariana.

**Reply by adamrher** [reply]
@fischer-ncar do you have a core hour p simulated year for the i case?

[edit - also, when you undersubscribe, does it tie up the entire node? I'm inferring that your using 14,336/224 = 64 processors per node.]

**Reply by fischer-ncar** [reply]
@adamrher I've only been running for 3 timesteps.  So I don't have any useful core hour p simulated year.

Yes, I'm tying up the entire node.
